entity-framework/core/get-started/xamarin.md

```diff
@@ -187,6 +187,23 @@ using (var blogContext = new BloggingContext())
 }
 ```
 
+## iPhone Configuration
+
+Entity Framework Core sometimes generates dynamic code for performance reasons. This is not supported on the iOS platform and may cause the application to crash. The Xamarin team built an interpretor to handle these scenarios. It should be configured in your `.csproj` file in a `PropertyGroup`. Use a `Condition` to ensure the group is only valid for iPhone release builds:
+
+```xml
+<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|iPhone' ">
+</PropertyGroup>
+```
+
+Then add the `--interpreter` flag as an extra argument inside the group:
+
+```xml
```

Comment:
Combining both xml blocks would make it clearer what is happening overall. Also easy to copy paste.